### PR TITLE
Add result of investigation on the needed HTTP APIs

### PR DESCRIPTION
**Pull request by victornoel**
This is for #64.

### CODE REVIEW

**Comment by victornoel** [comment]
@g4s8 FYI

**Comment by victornoel** [comment]
@g4s8 it seems the windows build is broken and it's not related to this PR.

**Comment by 0crat** [comment]
This pull request [#67](https://github.com/artipie/maven-adapter/issues/67) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/3cb95085-91ca-4e31-8c89-0804b03bce35); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by victornoel** [comment]
@g4s8 ping (the build is not passing because of #70)

File: README.md [hunk 1]
```diff
@@ -27,7 +27,30 @@ To avoid build errors use Maven 3.2+.
...
 ## How it works
 
-## Files
+When the Maven client downloads a dependency, it relies on normal HTTP GET based on well-known files (see below for details).
+
+In correspondance with this, Artipie responds to HTTP requests for each of the type of files like a normal HTTP server.
+
+<!--
+@todo #64:30min Continue working in the Maven HTTP API for downloading artifacts: we need to 1) add
+IT tests that uses a traditional Maven client (investigate for libraries used in the Maven
```
Comment (g4s8) [inline README.md:36]: @victornoel there are should be one space indentation for todo marker. The format is:
```txt
@todo #1:30min First line
 second line
 third line
 more lines...
```

File: README.md [hunk 2]
```diff
@@ -27,7 +27,30 @@ To avoid build errors use Maven 3.2+.
...
+so that we serve the artifacts according to the layout described below.
+-->
+
+When the Maven client uploads a dependency, it relies on the maven-deploy-plugin being configured, a
+repository being configure in the `distributionManagement` section of the `settings.xml` and authentication
+credentials configured in the `servers` section of the`settings.xml`.
+
+<!--
+@todo #64:30min Continue working in the Maven HTTP API for uploading artifacts: we need to 1) investigate
+exactly what are the HTTP endpoints needed when using the maven-deploy-plugin with the HTTP Wagon extension,
```
Comment (g4s8) [inline README.md:47]: @victornoel same here

**Comment by victornoel** [comment]
@g4s8 sorry about that, now it should be ok

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Pull request merged**